docs/docs/dev/explore/overview.md

```diff
@@ -1,128 +1,84 @@
 # Overview
 
-As we have [already learned](../documentation/introduction.md), Kuma is a universal control plane that can run across both modern environments like Kubernetes and more traditional VM-based ones.
+This sections gives an overview of a Kuma service mesh.
+It also covers how to start integrating your services into your mesh.
 
-The first step is obviously to [download and install Kuma](/install/) on the platform of your choice. Different distributions will present different installation instructions that follow the best practices for the platform you have selected.
+A Kuma mesh is made up of two main components:
 
-Regardless of what platform you decide to use, the fundamental behavior of Kuma at runtime will not change across different distributions. These fundamentals are important to explore in order to understand what Kuma is and how it works.
+- **Data plane**: The data plane consists of your services
```

Comment:
Is that true? I think the most common definition is that the service is outside of the data plane - see Istio's arch https://istio.io/latest/docs/ops/deployment/architecture/arch.svg

Reply:
Yeah gonna fix this